include/quic/parser.hpp

```diff
@@ -0,0 +1,384 @@
+#include <oxenc/bt.h>
+
+#include "stream.hpp"
+#include "utils.hpp"
+
+namespace oxen::quic
+{
+    inline auto bp_cat = oxen::log::Cat("bparser");
+
+    using time_point = std::chrono::steady_clock::time_point;
+
+    // timeout is used for sent requests awaiting responses
+    const std::chrono::seconds TIMEOUT{10};
+
+    // request sizes
+    inline constexpr long long MAX_REQ_LEN{10000000};
+
+    // Application error
+    inline constexpr uint64_t BPARSER_EXCEPTION = (1ULL << 60) + 69;
+
+    inline std::basic_string<std::byte> operator""_bs(const char* __str, size_t __len) noexcept
```

Comment:
Leading-double-underscore names are reserved for internal use by the c/c++ compiler.  (Also, leading-single-underscore-followed-by-a-capital-letter).